src/app/components/SubmitButton/SubmitButton.tsx

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
 import type { ReactNode } from 'react';
 import styles from './SubmitButton.module.css';
+import Typography from '../Typography/Typography';
 
 type SubmitButtonProps = {
+  icon:
+    | 'ArrowRight'
+    | 'Camera'
+    | 'Letter'
+    | 'Safe'
+    | 'Search'
+    | 'Send'
+    | 'Signup'
+    | 'Upload';
```

Comment:
I understand why you capitalized these, but it looks weird. If you aren't using components as icons, you shouldn't capitalise in this case.